src/DI/Loader/DoctrineAnnotationLoader.php

```diff
@@ -124,15 +124,15 @@ protected function analyseClass($class)
 	 */
 	protected function acceptController(ClassType $class)
 	{
-		// Has class annotation @Controller?
-		if ($class->hasAnnotation('Controller')) return TRUE;
+		// Has class annotation @Controller?	// Has class annotation @Controller?
+		if ($this->createReader()->getClassAnnotation($class, \Apitte\Core\Annotation\Controller\Controller::class)) return TRUE;
```

Comment:
Please import class ;-)

Reply:
The class cannot be imported. There is already import for
 ```use Apitte\Core\Schema\Builder\Controller\Controller;```

Maybe alias Controller annotation?

Reply:
Alias would be great. Thanks.